docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -0,0 +1,144 @@
+    process
+
+If Teleport Identity is enabled, the enrolment process will also guide you
+through the process of enabling SCIM provisioning for the Okta SAML application.
+
+<Admonition type="note">
+If a SAML Connector called `okta-integration` already exists, the enrolment
+process will attempt to adopt this connector and use the Okta SAML Application
+it points to, rather than create a new one.
+
+This adoption process will fail if the SAML Connector was created with Teleport 
+versions prior to 15.0.0, or was created manually. In this case, the best course
+of action is to either delete or rename the existing SAML Connector and let the
+integration enrolment create a new one.
+</Admonition>
+
+#### How to enrol
+
+Visit the Teleport Web UI and click **Access Management** on the menu bar at the
+top of the screen. 
+
+On the left sidebar, click **Enroll New Integration** to visit the 
+"Enroll New Integration" page:
+
+![Enroll an Access Request plugin](../../../img/enterprise/plugins/enroll.png)
+
+Select the Okta tile, and then Teleport will then ask for  
+ * your Okta organization URL, and 
+ * the OKta API token described above.
+ 
+Click **Connect** to continue.
+
+Once the initial enrolment has completed, and if Teleport Identity
+is enabled, the enrolment workflow will guide you through enabling SCIM
+provisioning as well.
+
+#### Configuring SCIM provisioning
+
+If Teleport Identity is enabled, the enrolment workflow will now show you a
+guide to configuring SCIM. 
+
+To recap the enrolment "Set Up SCIM" guide:
+
+##### Enable SCIM Provisioning
+ 1. In the Okta Admin Console, go to **Applications** and select the Teleport-
+    created application, or use the link supplied by the Teleport "Set Up SCIM" 
+    guide.
+ 1. Click **Edit** on the Okta **App Settings** page, check the box marked
+    **Enable SCIM provisioning**, and click **Save**.
+
+##### Configure Okta SCIM Client
+ 1. Click on the new **Provisioning** tab and then click **Edit**.
+ 1. Copy the **SCIM connector base URL** from the Teleport **Set Up SCIM** guide
+    and paste it into **SCIM connector base URL** field in the Okta provisioning
+    page.
+ 1. Set the Okta **Unique identifier field for users** to `userName`.
+ 1. Under **Supported provisioning actions**, check the following boxes:
+    1. Import New Users and Profile Updates
+    1. Push New Users
+    1. Push Profile Updates
+ 1. Set **Authentication Mode** to `HTTP Header`
+ 1. Copy the authorization **Bearer Token** from the Teleport **Set Up SCIM**
+    guide and paste it into **Bearer Token** field in the Okta provisioning
+    page.
+ 1. Click **Save**
+
+##### Configure Okta-To-App Provisioning
+ 1. Staying on the **Provisioning** tab, go to the new **To App** Okta
+    application settings page and click **Edit**.
+ 1. Check the following checkboxes:
+    1. Create Users
+    1. Update User Attributes
+    1. Deactivate Users
+ 1. Click **Save**
```

Comment:
Can we provide screenshots flow examples ? I think that it will be very helpful for a user.